doc/modules/model_evaluation.rst

```diff
@@ -2694,6 +2694,38 @@ Here are some usage examples of the :func:`d2_absolute_error_score` function::
   >>> d2_absolute_error_score(y_true, y_pred)
   0.0
 
+Visual evaluation of regression models
+--------------------------------------
+
+The :class:`~sklearn.metrics.PredictionErrorDisplay` displays allows to visually
+inspect the quality of regression models. The quality of a regression model can
```

Comment:
Maybe mention that it's not applicable for quantile regression as Christian pointed out